vividus-facade-jira/src/main/java/org/vividus/jira/JiraFieldsMappingProvider.java

```diff
@@ -0,0 +1,22 @@
+package org.vividus.jira;
+
+import java.util.Map;
+import java.util.Optional;
+
+import org.vividus.jira.model.JiraConfiguration;
+
+public class JiraFieldsMappingProvider
```

Comment:
is it really needed? what's the profit to keep `JiraConfigurationProvider` package-private?

Reply:
JiraConfiguration is a mutable internal thing that contains configs that's why I think it should not be exposed as API provided to end-users and this is the reason why I created safe JiraFieldsMappingProvider

Reply:
JiraConfigurationProvider.getFieldsMappingByProjectKey

Reply:
hmm okay, this is better as long as we have only one method to retrieve fields mapping and not much of business logic